src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/best-practices/opentelemetry-best-practices-browser.mdx

```diff
@@ -0,0 +1,21 @@
+---
+title: 'OpenTelemetry JavaScript and Browser: Best practices'
+tags:
+  - Integrations
+  - Open source telemetry integrations
+  - OpenTelemetry
+metaDescription: Here are some tips for working with OpenTelemetry, browser applications, and New Relic.
+---
+
+<Callout variant="caution">
+We recommend that you do not send telemetry data from a browser application or other client-side code directly to New Relic's OTLP endpoint because this could expose your license key.
```

Comment:
I would not say "other client-side code" because mobile instrumentation is client-side and does not expose the license key. I would recommend either removing this or clarifying, e.g. "... or any other client-side code that exposes the license key..."